Makefile

```diff
@@ -1,8 +1,30 @@
+GOCMD=go
+GOBUILD = $(GOCMD) build
+GOCLEAN = $(GOCMD) clean
+GOTEST  = $(GOCMD) test
+GOGET   = $(GOCMD) get
+GOTOOL  = $(GOCMD) tool
+GOBASE := $(shell pwd)
+GOBIN  := $(GOBASE)/bin
+DOCKER := $(GOBASE)/tools/test
+
 # name of executable.
-BINARY=pipeline
+BINARY = pipeline
 
 include skeleton/pipeline.mk
 
+clean: clean-containers
+	@echo "  >  Cleaning binaries and cache"
+	@-rm -f $(GOBIN)/$(PROJECTNAME)/$(BINARY)
+	@$(GOCLEAN)
+
+clean-containers:
+	@echo "  >  Cleaning containers"
+	@cd $(DOCKER) && docker-compose kill && docker-compose rm -f
```

Comment:
Can simplify to `docker-compose down` ([doc](https://docs.docker.com/compose/reference/down/)).

Reply:
Done